provers/celestia-prover/README.md

```diff
@@ -0,0 +1,27 @@
+# Celestia Prover
+
+The Celestia Prover is a gRPC service that generates zero-knowledge proofs for Celestia state transitions and data membership. It is designed to work with IBC (Inter-Blockchain Communication) and specifically implements proofs compatible with the ICS-07 Tendermint client specification.
+
+## Usage
+
+> ⚠️ **Warning**
+> This gRPC service is still under development and may not work as described
```

Comment:
[optional] GitHub now supports a special syntax to make this warning pretty in GitHub UI. 

```suggestion
> [!WARNING] 
> This gRPC service is still under development and may not work as described
```

Ref: https://github.com/orgs/community/discussions/16925